evap/evaluation/models.py

```diff
@@ -414,7 +422,7 @@ def days_left_for_evaluation(self):
 
     @property
     def days_until_evaluation(self):
-        return (self.vote_start_date - datetime.date.today()).days
+        return (self.vote_start_date - datetime.datetime.now()).days
```

Comment:
as per @janno42's request, can you change this to only take days into account, not the time?